ecosystem/indexer-grpc/indexer-grpc-utils/src/file_store_operator_v2/common.rs

```diff
@@ -0,0 +1,44 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::Result;
+use serde::{Deserialize, Serialize};
+use std::path::PathBuf;
+use tokio::time::Duration;
+
+pub const METADATA_FILE_NAME: &str = "metadata.json";
+
+#[derive(Serialize, Deserialize, Debug)]
+pub struct FileStoreMetadata {
+    pub chain_id: u64,
+    pub num_transactions_per_folder: u64,
+    pub version: u64,
+}
+
+#[derive(Serialize, Deserialize, Default, Clone)]
+pub struct BatchMetadata {
+    // "[first_version, last_version), size_bytes"
+    pub files: Vec<(u64, u64, usize)>,
+}
+
+#[async_trait::async_trait]
+pub trait IFileStoreReader: Sync + Send {
+    /// The tag of the store, for logging.
+    fn tag(&self) -> &str;
+
+    async fn is_initialized(&self) -> bool;
```

Comment:
need comments: what does `true` and `false` stand for?

Reply:
Done